x/feegrant/keeper/genesis_test.go

```diff
@@ -145,3 +145,40 @@ func TestInitGenesis(t *testing.T) {
 		})
 	}
 }
+
+func TestDuplicateGrantsInGenesis(t *testing.T) {
+	f := initFixture(t)
+
+	addrCdc := f.accountKeeper.AddressCodec()
+	granter, err := addrCdc.BytesToString(granterAddr.Bytes())
+	assert.NilError(t, err)
+	grantee, err := addrCdc.BytesToString(granteeAddr.Bytes())
+	assert.NilError(t, err)
+
```

Comment:
this is not necessary, `granteeAddr` is already an `sdk.AccAddress` which can be directly converted to a string